### ISSUE TITLE
Notification on game mention

### ISSUE THREAD

**Issue by Cerno-b** [ludumdare/ludumdare#1538]
Not sure this is implemented already, but I think it isn't:

So far you get a notification when your username is tagged.
However I think it would be pretty cool if **you could get a notification when your game is mentioned** somewhere, either in the feed, or in a comment.

I guess there would have to be some more parsing involved than looking for a username, but it should be possible, right?

**Comment by local-minimum** [comment]
Yeah nothing like it. It become a bit problematic to parse all new nodes against all published games. Especially if you don't want to require perfect match like capitalization, punctuation.  It should be possible though,  but we with a lot of work. Linking to your game or post may be more tangible and likely to capture most of what you had in mind.

**Comment by Cerno-b** [comment]
Actually that's exactly what I had in mind, sorry for wording it poorly. I didn't want a notification when the name of your game is dropped, but when a link to your game is dropped.

**Comment by mikekasprzak** [comment]
That sounds more doable, but in practice I'm not sure if it'll get used
that often (pretty much only when folks make list posts).

**Comment by local-minimum** [comment]
Yeah I figure that is the use case. Still if someone posted "This is my top 5 games that you probably missed" I would really like to see it even if they didn't think about at:ing me (and if "me" is a team they most probably only at:ed the main author).

**Comment by Cerno-b** [comment]
This is my main use case as well. It's very hard to miss if someone likes your game so much that they put it in a feed post and I would people to have the opportunity to witness this :)

**Comment by mikekasprzak** [comment]
Well, on the agenda is an actual list making feature (wishlist/play later
lists, custom lists). At the moment there is no concept of publishing
lists, but lists themselves are nodes, so it's not much of a stretch. If
nothing else, on-list-publish is an opportunity to add another
notification.
